install_config/install/prerequisites.adoc

```diff
@@ -223,8 +277,48 @@ domain is not listed in the search list. Otherwise, containers managed by
 OpenShift may fail to resolve host names properly.
 ====
 
-[[prereq-network-access]]
 
+[[dns-config-prereq]]
+*Configuring a DNS Environment*
+
+To properly configure your DNS environment for {product-title}:
+
+. Ensure that the DNS servers listed in *_/etc/resolv.conf_* are able to resolve the addresses of all the masters and nodes in your OpenShift environment:
++
+----
+$ cat /etc/resolv.conf
+$ dig <node_hostname> @<IP_address> +short
+----
++
+For example:
++
+----
+$ cat /etc/resolv.conf
+search example.com
+nameserver 172.16.25.35
+
+$ dig ose3-node1.example.com  @172.16.25.41 +short
+172.16.25.45
+----
++
+. If DHCP is:
++
+- Disabled, then configure your network interface to be static.
+- Enabled, then the NetworkManager Dispatch script configures DHCP automatically. Optionally you can configure an IP address to be prepended to the *_/etc/resolve.conf_* of a pod by adding a value to the `*dnsIP*` option of the *_node-config.yaml_* file. By default, this will be the IP address of the node host.
```

Comment:
NetworkManager Dispatch script configure DNS based off what is set by DHCP.   

Misleading: 
"Optionally you can configure an IP address to be prepended to the *_/etc/resolve.conf_* of a pod by adding a value to the `*dnsIP*` option of the *_node-config.yaml_* file. By default, this will be the IP address of the node host."

This will not change your *_/etc/resolve.conf_* on the host it will only prepend the pods resolve.conf and the 2nd nameserver will be taken from what the host has set for the its first nameserver.